build/ical.js

```diff
@@ -1073,7 +1073,7 @@ ICAL.design = (function() {
     "nickname": DEFAULT_TYPE_TEXT_MULTI,
     "note": DEFAULT_TYPE_TEXT,
     "org": { defaultType: "text", structuredValue: ";" },
-    "photo": DEFAULT_TYPE_URI,
+    "photo": { defaultType: "binary", allowedTypes: ["binary", "uri"] },             //XXXgW
```

Comment:
If this is the right change, then please remove XXX. If it is temporary, please remove it.